portal_pages/templates/portal_pages/blog_page_add.html

```diff
@@ -0,0 +1,145 @@
+{% extends "base.html" %}
+{% load static %}
+{% load wagtailcore_tags %}
+
+{% block extra-header %}
+    <script type="text/javascript" src="{% static 'js/tinymce/tinymce.min.js' %}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script type="text/javascript" src="{% static 'js/chosen/chosen.jquery.min.js' %}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
+{% endblock %}
+
+{% block extra-css %}
+    <link href="{% static 'css/chosen.css' %}" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+{% endblock %}
+
+{% block content %}
+<div class="page-width float-layout"><div class="cushion">
+
+    <form enctype="multipart/form-data" action="." method="post" onsubmit="return checkFormAndSubmit();">
+    {% csrf_token %}
+
+    <div class="two-thirds"><div class="cushion">
+
+        <div class="cushion">
+            {{ blog_index.submit_info|richtext }}
+        </div>
+
+        <div class="cushion">
+            {% if form.errors %}
+                <p class="cushion errorlabel">Please correct the errors below.</p>
+            {% endif %}
+
+            {{ form.honeypot }}
+
+            {{ form.media }}
+            {{ form.media.errors }}
```

Comment:
Same comments as on case study...honeypot better at bottom, media should be in js-extra block, and no need to render media.errors...